packages/rainbowkit/src/components/ConnectModal/ConnectModalIntro.tsx

```diff
@@ -18,92 +19,91 @@ export function ConnectModalIntro({
 }) {
   const { disclaimer: Disclaimer, learnMoreUrl } = useContext(AppContext);
 
+  const i18n = useContext(I18nContext);
+
   return (
-    <>
+    <Box
+      alignItems="center"
+      color="accentColor"
+      display="flex"
+      flexDirection="column"
+      justifyContent="space-around"
+      flex="1"
+      overflowY="auto"
+    >
+      <Box marginBottom="10">
```

Comment:
Ignore a lot of these changes. We used to have some `height` and `min-height` tricks to make rainbowkit look suitable but right now it doesn't support responsive design. So i use flexbox and overflow to make this work 🙏

Example:

<img width="741" alt="image" src="https://github.com/rainbow-me/rainbowkit/assets/53529533/44710873-a26a-48dc-8b55-551cb1a16b04">

Reply:
Nice 👌